instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/rum/RumConfigE2ETests.kt

```diff
@@ -303,7 +303,8 @@ class RumConfigE2ETests {
                     logsEnabled = true,
                     tracesEnabled = true,
                     rumEnabled = true,
-                    crashReportsEnabled = true
+                    crashReportsEnabled = true,
+                    sessionReplayEnabled = true
```

Comment:
why do we depend on the session replay feature here and below?

Reply:
We do not, I just followed the pattern on the other features.